utils/mamUtils.js

```diff
@@ -173,9 +172,6 @@ class mam {
     if (recvPub.length !== 1) { console.log('error: Get initial claim failed'); }
     recvPub = recvPub[0].message.pk;
 
-    const sendPriv = this.accountStore.findOnly({ id: params.sender }).sk;
```

Comment:
The statement is not just assignment, it also makes a query to the Loki data store.

Since the return values are never used, the query in here is unnecessary. 🤔

Reply:
Any tools to track or detect such query invocation?

Reply:
From my knowledge, there is no tool to detect this problem. 🤔